matching-service/frontend/src/components/RoomPage/RoomPage.js

```diff
@@ -18,10 +27,10 @@ export default function RoomPage() {
 
```

Comment:
I don't think you need `socket.disconnect()` after emitting leave-room since the server, in this case, will take care of disconnecting